### ISSUE TITLE
[HOLD] Hitting "Enter" won't confirm the selected currency

### ISSUE THREAD

**Issue by aman-atg** [Expensify/App#4496]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Login to newDot
2. Go to any chat
3. Select Request Money option
4. Click on the currency
5. Select any currency

## Expected Result:
User should be able to confirm a currency using "Enter"

## Actual Result:
Nothing happened.

https://user-images.githubusercontent.com/28243942/128604041-0dfaeb7e-20a4-4010-bdcf-c68a9cb303ac.mp4

## Workaround:
User can hit Confirm using cursor.

## Platform:

Where is this issue occurring?

- Web
 

**Version Number:**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @MitchExpensify (`AutoAssignerTriage`), see https://stackoverflow.com/c/expensify/questions/4749 for more details.

**Comment by aman-atg** [comment]
## Proposal 
* We can fix this by passing `pressOnEnter` to `Button`
https://github.com/Expensify/App/blob/50828ab6752324988fbfdc7020878c80adf174bd/src/pages/iou/IOUCurrencySelection.js#L214-L220

**Comment by MelvinBot** [comment]
@MitchExpensify Whoops! This issue is 2 days overdue. Let's get this updated quick!

**Comment by mallenexpensify** [comment]
Putting on hold for week-ish while we focus on internal priorities.

**Comment by mallenexpensify** [comment]
Coming back to this... I'm unsure if it's worth fixing, at least from a stand-alone action.  In order to change the currency you already have to navigate using a mouse/trackpad/ointer, so it's not 'unexpected' that you'd also have to use the mouse to also confirm the selection.  Also, changing currencies is likely going to happen frequently. 

I'm a HUGE fan of keyboard shortcuts.  A month or two back a contributor found a similar issue (tapping enter completed an action) then went through and found many other examples, we bundled those together for a fix.  I searched for the issue but was unable to find. 

Maybe put on hold and add more examples over time if we come to them? We could update the title to 'Hitting 'enter' should do (insert action)" or.. something _better_.
